demos/fleet_provisioning/fleet_provisioning_csr/demo_config.h

```diff
@@ -0,0 +1,215 @@
+ * directory from @ref https://www.amazontrust.com/repository/AmazonRootCA1.pem
+ * using the CMake build system.
+ *
+ * @note This certificate should be PEM-encoded.
+ * @note This path is relative from the demo binary created. Update
+ * ROOT_CA_CERT_PATH to the absolute path if this demo is executed from elsewhere.
+ */
+#ifndef ROOT_CA_CERT_PATH
+    #define ROOT_CA_CERT_PATH    "certificates/AmazonRootCA1.crt"
+#endif
+
+/**
+ * @brief Path the file containing the provisioning claim certificate. This
+ * certificate is used to authenticate with AWS IoT for provisioning by claim.
+ *
+ * For information about provisioning by claim, see the following AWS documentation:
+ * https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html#claim-based
+ *
+ * @note This certificate should be PEM-encoded.
+ *
+ * #define CLAIM_CERT_PATH    "...insert here..."
+ */
+
+/**
+ * @brief Path the file containing the provisioning claim private key. This
+ * key corresponds to the provisioning claim certificate and is used to
+ * authenticate with AWS IoT for provisioning by claim.
+ *
+ * For information about provisioning by claim, see the following AWS documentation:
+ * https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html#claim-based
+ *
+ * @note This private key should be PEM-encoded.
+ *
+ * #define CLAIM_KEY_PATH    "...insert here..."
+ */
+
+/**
+ * @brief Path of the file containing the private key used for provisioning. This should be the key
+ * used to generate the Certificate Signing Request (CSR) and is used to connect to IoT Core with
+ * the provisioned certificate.
+ *
+ * For information about provisioning by claim, see the following AWS documentation:
+ * https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html#claim-based
+ *
+ * @note This private key should be PEM-encoded.
+ *
+ * #define PROVISIONING_KEY_PATH    "...insert here..."
+ */
+
+/**
+ * @brief Path of the file containing the Certificate Signing Request (CSR) to send to the AWS IoT
+ * Fleet Provisioning APIs for provisioning the new certificate.
+ *
+ * For information about provisioning by claim, see the following AWS documentation:
+ * https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html#claim-based
+ *
+ * @note This CSR should be PEM-encoded.
+ *
+ * #define PROVISIONING_CSR_PATH    "...insert here..."
+ */
+
+/**
+ * @brief Path to which to write the newly provisioned client certificate obtained from AWS
+ * IoT Fleet Provisioning.
+ **
+ * For information about provisioning by claim, see the following AWS documentation:
+ * https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html#claim-based
+ *
+ * For information about client certificates, see the following AWS documentation:
+ * https://docs.aws.amazon.com/iot/latest/developerguide/x509-client-certs.html
+ *
+ * @note This certificate should be PEM-encoded.
+ *
```

Comment:
This certificate will be provided by AWS IoT, so this `@note` can be confusing as instructing the user about a cert they are providing the demo
```suggestion
```